### PR TITLE
fixed flash messages error for issue states and issue types

### PR DESCRIPTION
**Pull request by abubakarazeem**

### CODE REVIEW

**Comment by abubakarazeem** [comment]
review: cycle 0

File: app/views/issue_states/update.js.erb [hunk 1]
```diff
@@ -1,9 +1,7 @@
-issueStateFlash.html('<%= j render partial: "layouts/notice" %>');
+issueStateFlash.html('<%= j render partial: "shared/flash" %>');
 <% if @issue_state.errors.blank? %>
   tr = $("#<%= issue_state_table_row_id(@issue_state) %>");
   tr.html('<%= j render partial: 'issue_state_table_row_body', locals: {issue_state: @issue_state} %>');
   issueStatesTable.row(tr).invalidate().draw();
-<% else %>
-  issueStatesErrors.html('<%= j render partial: "layouts/errors", locals: {model: @issue_state} %>');
```
Comment (basil7vals) [inline app/views/issue_states/update.js.erb]: locals: {issue_state: @issue_state}

File: app/views/issue_states/create.js.erb [hunk 2]
```diff
@@ -1,9 +1,7 @@
-issueStateFlash.html('<%= j render partial: "layouts/notice" %>');
+issueStateFlash.html('<%= j render partial: "shared/flash" %>');
 <% if @issue_state.errors.blank? %>
   $("#issue_state_form").trigger('reset');
   issueStatesTableBody.append('<%= j render partial: 'issue_state_table_row', locals: {issue_state: @issue_state} %>');
   tr = $("#<%= issue_state_table_row_id(@issue_state) %>");
   issueStatesTable.row.add(tr).draw();
-<% else %>
-  issueStatesErrors.html('<%= j render partial: "layouts/errors", locals: {model: @issue_state} %>');
```
Comment (basil7vals) [inline app/views/issue_states/create.js.erb]: locals: {issue_state: @issue_state}

File: app/views/issue_states/update.js.erb (hunk 1 above)
Comment (abubakarazeem) [reply app/views/issue_states/update.js.erb]: removed this line

File: app/views/issue_states/create.js.erb (hunk 2 above)
Comment (abubakarazeem) [reply app/views/issue_states/create.js.erb]: this line has been removed

**Comment by basil7vals** [comment]
Cycle - 1 completed

**Comment by basil7vals** [comment]
Cycle - 1 completed

**Pull request merged**